config.sample.yaml

```diff
@@ -82,4 +83,7 @@ bindings:
   close_modal: ["<esc>"]
 
   # the key binding to close the application.
-  exit: ["<c-c>"]
+  exit: ["<c-c>", "q"]
```

Comment:
Yeah I'm not sure about this. vim uses `:q<enter>` which would be okay, but this is too easy to accidentally press as it is. You can always define this in your custom config file.

Reply:
@mfontanini, that's true and perhaps not the best example. But using just `q` is common enough in other programs---`lookatme` uses `q` and so does [`lazygit`](https://github.com/jesseduffield/lazygit) and others